namespaces/systemd.yml

```diff
@@ -194,3 +259,108 @@ namespace:
       type: string
       description: >
         TODO
+
+    - name: CONTAINER_ID
+      type: string
+      description: >
+        TODO
+
+    - name: CONTAINER_ID_FULL
+      type: string
+      description: >
+        TODO
+
+    - name: CONTAINER_NAME
+      type: string
+      description: >
+        TODO
+
+    - name: CONTAINER_TAG
+      type: string
+      description: >
+        TODO
+
+    - name: EXIT_CODE
+      type: string
+      description: >
+        TODO
+
+    - name: EXIT_STATUS
+      type: string
+      description: >
+        TODO
+
+    - name: LEADER
+      type: string
+      description: >
+        TODO
+
+    - name: LIBVIRT_CODE
+      type: string
+      description: >
+        TODO
+
+    - name: LIBVIRT_DOMAIN
+      type: string
+      description: >
+        TODO
+
+    - name: LIBVIRT_SOURCE
+      type: string
+      description: >
+        TODO
+
+    - name: NM_LOG_DOMAINS
+      type: string
+      description: >
+        TODO
+
+    - name: NM_LOG_LEVEL
+      type: string
+      description: >
+        TODO
+
+    - name: PACKAGE
+      type: string
+      description: >
+        TODO
+
+    - name: SESSION_ID
+      type: string
+      description: >
+        TODO
+
+    - name: SSSD_DOMAIN
+      type: string
+      description: >
+        TODO
+
+    - name: TIMESTAMP_BOOTTIME
+      type: string
+      description: >
+        TODO
+
+    - name: TIMESTAMP_MONOTONIC
+      type: string
+      description: >
+        TODO
+
+    - name: USERSPACE_USEC
+      type: string
+      description: >
+        TODO
+
+    - name: USER_ID
+      type: string
+      description: >
+        TODO
+
+    - name: USER_UNIT
+      type: string
+      description: >
+        TODO
+
+    - name: SOURCE_MONOTONIC_TIMESTAMP
+      type: string
```

Comment:
This field and many others are type `string` - should they be `int` or `double` or some other numeric field type?